docker/init_db.sql

```diff
@@ -12,9 +12,12 @@
 -- limitations under the License.
 --
 
-ALTER USER pulsar WITH PASSWORD 'pulsar';
-CREATE DATABASE pulsar_manager OWNER pulsar;
-GRANT ALL PRIVILEGES ON DATABASE pulsar_manager to pulsar;
+\set username `echo "$USERNAME"`
+\set password `echo "$PASSWORD"`
```

Comment:
I wonder if it would be worth preserving backwards compatibility with a default value for USERNAME and PASSWORD?
```suggestion
\set username `echo "${USERNAME:-pulsar}"`
\set password `echo "${PASSWORD:-pulsar}"`
```